### PR TITLE
Updated README [ciskip]

### LINKED ISSUE #96: README review
## Notes

Notes subsections below describe the corresponding subsections in the readme.

### Features

> Runs tests built using @dojo/cli-build-webpack, or @dojo/cli-build-app.

No comma

### How do I use this package?

> There are several configuration options, to see what options are available run dojo test -h

Use a semicolon rather than a comma, or make these separate sentences.

### How do I contribute?

Maybe _How can I contribute?_

> An additional npm script to run prettier (with write set to true) against all src and test project files is available by running:

Maybe better worded as You format all src and test project files by running:

### Testing

> Test cases MUST be written using Intern using the bdd test interface and Assert assertion interface.

_...the "bdd" test interface and "assert" assertion interface._

### Licensing information

We don't normally list out the licenses of dependent packages.

Update the copyright line to 2018, and start at the real start year, not 2004.

### PR DESCRIPTION
**Pull request by devpaul**
Adds updated usage and information for running tests on testing platforms

Resolves #96

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,19 +5,118 @@
...
+The official Dojo 2 test command. This package uses Intern to run unit and functional tests against your Dojo project.
 
-**WARNING** This is _beta_ software. While we do not anticipate significant changes to the API at this stage, we may feel the need to do so. This is not yet production ready, so you should use at your own risk.
+## Usage
 
-## Features
+### Prerequisites
 
-Runs tests built using `@dojo/cli-build-webpack`, or `@dojo/cli-build-app`.
+This project is a plugin for the Dojo CLI. Please visit the [Dojo CLI] project for
```
Comment (agubler) [inline README.md]: Can we add a link in here? And swap `plugin` for `command`
```
[Dojo CLI](https://github/dojo/cli)
```

File: README.md [hunk 2]
```diff
@@ -5,19 +5,118 @@
...
+ information about the project and how to install.
 
+### Installation
 
-## How do I use this package?
+The use `@dojo/cli-test-intern` in a project, install the packages:
```
Comment (agubler) [inline README.md]: "install the packages:" -> "install the package:"

File: README.md [hunk 3]
```diff
@@ -5,19 +5,118 @@
...
+
+### Functional tests
+
+Functional tests may be run explicitly with the `-f` flag or as part of a full test run using the `-a` flag
+
+```bash
+dojo test -f
+```
+
+cli-test-intern will execute tests located at `./output/test/functional.js` in Chrome using Selenium and provide a
```
Comment (agubler) [inline README.md]: cli-test-intern -> `@dojo/cli-test-intern`

File: README.md [hunk 4]
```diff
@@ -5,19 +5,118 @@
...
+
+### Unit tests
+
+Unit tests may be run explicitly with the `-u` flag or as part of a full test run using the `-a` flag
+
+```bash
+dojo test -u
+```
+
+cli-test-intern will execute tests located at `output/tests/unit.js` in node and in Chrome and provide a report
```
Comment (agubler) [inline README.md]: cli-test-intern -> `@dojo/cli-test-intern`

File: README.md (hunk 3 above)
Comment (agubler) [reply README.md]: And else where in the readme also

File: README.md [hunk 5]
```diff
@@ -5,19 +5,118 @@
...
+```bash
+dojo test -f
+```
+
+cli-test-intern will execute tests located at `./output/test/functional.js` in Chrome using Selenium and provide a
+ report listing any failed tests.
+
+### Testing services
+
+Intern comes with support for running tests remotely on [BrowserStack], [SauceLabs], and [TestingBot]. You may use one
```
Comment (agubler) [inline README.md]: Are you planning on adding links to BrowserStack, SauceLabs and TestingBot?

Comment (agubler) [reply README.md]: Ignore me they do link, didn't know that :)

File: README.md [hunk 6]
```diff
@@ -36,28 +135,21 @@ npm run prettier
...
-* [Glob](https://github.com/isaacs/node-glob)([ISC](https://opensource.org/licenses/ISC))
-* [grunt-tslint](https://github.com/palantir/grunt-tslint)([Apache 2.0](https://opensource.org/licenses/Apache-2.0))
-* [Sinon.JS](https://github.com/sinonjs/sinon)([New BSD](http://opensource.org/licenses/BSD-3-Clause))
-* [TSLint](https://github.com/palantir/tslint)([Apache 2.0](https://opensource.org/licenses/Apache-2.0))
-* [yargs](https://github.com/yargs/yargs)([MIT](https://opensource.org/licenses/MIT))
-
-© 2004–2017 JS Foundation & contributors. [New BSD](LICENSE) license.
+© 2018 [JS Foundation] & contributors. [New BSD](LICENSE) license.
 
+[@dojo/cli-build-app]: https://github.com/dojo/cli-build-app
```
Comment (agubler) [inline README.md]: I didn't know you could do this :)

File: README.md (hunk 5 above)
Comment (devpaul) [reply README.md]: They're linked at the bottom.

**Pull request merged**